test/client/scraper/test_general_metadata.py

```diff
@@ -28,6 +29,23 @@ def test_general(self):
             self.assertTrue(json_response['data']['images']['count'] > 0)
             self.assertEqual(json_response['status'], 200)
 
+    def test_general2(self):
```

Comment:
this seems a little redundant, can we instead do a for loop with the different test urls